packages/feedback-service/src/feedback-config/schema.ts

```diff
@@ -4,9 +4,15 @@ import {
 
 export const FeedbackConfigSchema: Schema = new Schema({
   appId: {
+    type: String,
+    required: false,
+    unique: false,
+  },
+  projectId: {
     type: String,
     required: true,
     unique: true,
+    default: 'home-spa',
```

Comment:
No need for a default value here. The projectId is auto generated, so this cannot be hard coded.\
Also, the projectId is mandatory when creating a config.
```suggestion
```